### DISCUSSION TITLE
http root directory

### DISCUSSION THREAD

**Discussion by tchandler48** [kej715/DtCyber#160] [Q&A]
I have nos2..8.7 865 up and running, slowly making changes.  I am trying to load some 
web pages so the http server will display them.  I am logging in as guest/guest
but I can not locate the root directory to put the pages so the server will display them,.

Any suggestions.

thank you
/tom c

**Comment by kej715** [comment]
Hi Tom,

The default root directory for the web server is the catalog of user WWW.
WWW's default password is WWWX. The server also allows users to create web
pages that can be accessed using URL's specifying usernames, like:

      http://yourserversaddress/~guest/index.html

In this case, files containing the web content need to be public or
permitted to user WWW.

cheers,
Kevin
